.vib/kibana/cypress/cypress/fixtures/testdata.json

```diff
@@ -1,9 +1,3 @@
 {
-  "dashboardTitle": "Test_dashboard",
-  "dashboardDescription": "Test_dashboard_description",
-  "visualizationTitle": "Test_visualisation",
-  "testIndexName": "test_index",
-  "remoteClusterName":"remoteCluster",
-  "remoteCluster":"localhost:90",
-  "workpadName":"testWorkpad"
+  "workpadName": "testWorkpad"
```

Comment:
Are you still using this file? I don't think so

Reply:
True! I forgot to delete it.